spec/models/restaurant_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Restaurant, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
There should be specs here. Remember to stick to the acceptance unit test cycle. 🤔